Cliqz/Privacy/UI/ControlCenter/OverviewViewController.swift

```diff
@@ -507,32 +507,34 @@ class OverviewViewController: UIViewController {
 	@objc private func trustSitePressed() {
         if !self.trustSiteButton.isSelected {
             self.delegate?.changeAll(state: .trusted, tableType: .page, completion: {
-                
+                self.delegate?.pauseGhostery(paused: false, time: Date())
```

Comment:
Can you move these lines into a new method and call it instead of duplicating the code multiple times?